### PR TITLE
Change story type

### PR DESCRIPTION
**Pull request by RamiroPinol**
Based on a talk with Alexis yesterday, we discussed about this feature request: [175295009](https://www.pivotaltracker.com/n/projects/2463484/stories/175295009).

Changing the type is not that easy for ongoing/finished stories because estimations and status are changed or deleted, so several validations need to be done and discussed.

The first MVP I'm working on is the possibility of select/change the type when story is in the unscheduled column, typically when you are creating it.

@mattvv What do you think of this first try? Still super buggy

### CODE REVIEW

**Review by mattvv — COMMENTED** [review]
I think approach looks good, well identified about losing estimations, maybe we should put a prompt and let the user decide if they want to lose that information, we can ask Alexis about that

**Comment by RamiroPinol** [comment]
MVP ready.
Known issue: Width of Geist-ui Select component can't be changed easily so card is now wider on unscheduled column

File: components/StoryCard.tsx [hunk 1]
```diff
@@ -95,10 +95,49 @@ const StoryCard = ({ story, state, index, addFilter }: StoryCardParams): JSX.Ele
...
+  useEffect(() => {
+    saveStoryType();
+  }, [type]);
+
   const openEstimationModal = (e): void => {
     e.stopPropagation();
     setIsModalVisible(true);
   };
+
+  const StoryTypeSelect = (): JSX.Element => {
```
Comment (mattvv) [inline components/StoryCard.tsx:120]: Let's extract this into a component, as we will likely want to use it on the StoryModal also

File: components/StoryCard.tsx [hunk 2]
```diff
@@ -95,10 +95,49 @@ const StoryCard = ({ story, state, index, addFilter }: StoryCardParams): JSX.Ele
...
+      <Select
+        placeholder="Type"
+        value={type}
+        size="mini"
+        disabled={story.id === 'pending'}
+        onChange={value => {
+          setType(value);
+          saveStoryType();
+        }}
+        width="80px">
```
Comment (mattvv) [inline components/StoryCard.tsx:131]: Did you try resizing with styled components?

**Review by mattvv — APPROVED** [review]
Looks good for now, left some feedback I think we can follow up with in a followup PR to add the type select to the StoryModal

File: components/StoryCard.tsx (hunk 2 above)
Comment (RamiroPinol) [reply components/StoryCard.tsx]: Yes, I tried without success

**Review by mattvv — APPROVED** [review]

**Pull request merged**